src/main/kotlin/no/nav/syfo/azuread/v2/AzureAdV2Client.kt

```diff
@@ -20,21 +20,50 @@ class AzureAdV2Client(
     private val httpClient: HttpClient,
     private val azureAdV2Cache: AzureAdV2Cache = AzureAdV2Cache()
 ) {
+
+    /**
+     * Returns a non-obo access token authenticated using app specific client credentials
+     */
+    suspend fun getAccessToken(
+        scope: String
+    ): AzureAdV2Token? {
+        return azureAdV2Cache.getOboToken(scope)
```

Comment:
Denne skjønte jeg ikke :/ Hvorfor vil vi returnere obotoken hvis det finnes i cache når vi egentlig skal hente clientsecrettoken..?

Reply:
Eller, det er kanskje bare navnet på metoden som henter ting ut av cachen som heter getOboToken? Kanskje den kan renames til getToken eller noe slikt så det ikke blir så forvirrende? 😅

Reply:
Godt poeng. Det er bare metoden i cachen som heter getOboToken, men enig i at den bør oppdateres for å reflektere at den også behandler OBO-tokens nå.